### PR TITLE
Fix anchor overlapping with text above in #2207

### LINKED ISSUE #2207: Anchor top-padding breaks user interaction
### Description

See https://github.com/fsharp/FAKE/pull/2202#issuecomment-439694422

### PR DESCRIPTION
**Pull request by JarnoNijboer**
### PR for solution provided in [#2202](https://github.com/fsharp/FAKE/pull/2202#issuecomment-439880023)

- fixes #2207

### CODE REVIEW

File: help/content/assets/css/style.css [hunk 1]
```diff
@@ -61,7 +61,14 @@ section.is-small {
 
 .content .anchor {
     text-decoration: none;
-    padding-top: 70px;
+}
+
+.content .anchor::before {
+    content: "";
+    display: block;
+    height: 50px;
```
Comment (florianbader) [inline help/content/assets/css/style.css]:
```suggestion
    height: 70px;
```

File: help/content/assets/css/style.css [hunk 2]
```diff
@@ -61,7 +61,14 @@ section.is-small {
...
+    margin: -50px 0 0;
```
Comment (florianbader) [inline help/content/assets/css/style.css]:
```suggestion
    margin: -70px 0 0;
```

**Comment by florianbader** [comment]
Looks good to me. I would increase the height and position to 70px to have a bit more space between the navigation and the text like we had before.

**Comment by JarnoNijboer** [comment]
At first when trying this out I started with `70px` for the height and margin. After fiddling I learned that the heigt/margin must be the same as the top navbar. Because that's the extra space that the content must be pushed down.

Image with height/margin of `50px`
image

And one with `70px`
image

**Comment by matthid** [comment]
@JarnoNijboer Thanks a lot for this detail work!

**Comment by baronfel** [comment]
Thanks y'all for finding the proper fix here

**Pull request merged**